autoload/leaderf/python/leaderf/devicons.py

```diff
@@ -78,6 +79,8 @@ def setAmbiwidth(val):
 # from vim-devicons
 # To use asynchronously
 def webDevIconsGetFileTypeSymbol(file):
+    if 'loaded_webdevicons' in vim.vars:
```

Comment:
I think it should be `if 'loaded_webdevicons' not in vim.vars:`

Reply:
I'm really sorry.
I fixed it.